src/components/Banner/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import BannerIcon from './components/BannerIcon';
+import BannerBody from './components/BannerBody';
+import BannerButton from './components/BannerButton';
+import BannerFooter from './components/BannerFooter';
+import BannerInfo from './components/BannerInfo';
+import BannerTitle from './components/BannerTitle';
+import { Props } from './types';
+
+
+function Banner({
```

Comment:
So this component being the highest level `Banner` component, it should probably have the ability to abstract away the other child components (Icon, Title, Footer, etc). We don't want to force the developer to construct every banner using minute components when we know the vast majority of use-cases will be the same.

Reply:
It should essentially abstract your documentation demo...
```jsx
<Banner
    color={color}
    variant={variant}
>
    <Banner.Icon iconName={iconName} isLoading={isLoading} />
    <Banner.Body>
        <Banner.Title>{title}</Banner.Title>
        <Banner.Info>{info}</Banner.Info>
    </Banner.Body>
</Banner>
```

Reply:
@lawren Doesn't this take away a lot of customization?